src/LinearCurve.sol

```diff
@@ -30,80 +30,64 @@ import {BaseCurve} from "./BaseCurve.sol";
  *      versions.
  */
 contract LinearCurve is BaseCurve {
-    /// @dev UD60x18 Max
-    uint256 public immutable MAX_SHARES = type(uint256).max / 1e18;
+    /// @dev Maximum number of shares that can be handled by the curve.
```

Comment:
3.15 - Unnecessarily Tight Limits in LinearCurve

This has been adjusted to the correct value, and is being checked by the EthMultiVault which is responsible for economic state flow.